File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/BuseoInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
	public class BuseoInfo
	{
		private String _buseoCode;
		private String _buseoName;

		public String BuseoCode
		{
			get { return this._buseoCode; }
			set { this._buseoCode = value; }
		}

		public String BuseoName
		{
			get { return this._buseoName; }
			set { this._buseoName = value; }
		}

		public BuseoInfo() { }

		public BuseoInfo(String buseoCode, String buseoName)
		{
			this._buseoCode = buseoCode;
			this._buseoName = buseoName;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/BuseoInfo.java
package nta.med.data.model.ihis.system;

public class BuseoInfo {
	private String buseoCode;
    private String buseoName;
	public BuseoInfo(String buseoCode, String buseoName) {
		super();
		this.buseoCode = buseoCode;
		this.buseoName = buseoName;
	}
	public String getBuseoCode() {
		return buseoCode;
	}
	public void setBuseoCode(String buseoCode) {
		this.buseoCode = buseoCode;
	}
	public String getBuseoName() {
		return buseoName;
	}
	public void setBuseoName(String buseoName) {
		this.buseoName = buseoName;
	}
}
